spec/factories/auth_source_factory.rb

```diff
@@ -36,4 +36,8 @@
     port 225  # a reserved port, should not be in use
     attr_login 'uid'
   end
+
+  factory :dummy_auth_source, class: DummyAuthSource do
+    name 'DerpLAP'
```

Comment:
I suggest you let the dummy auth source reside in the specs, i.e. create it in this Factory. It shouldn't be an actual model if  we can avoid it.